FESS-GUI/windowsnames.h

```diff
@@ -2,6 +2,12 @@
 #define WINDOWSNAMES
 
 #define MAINWINDOW_TITLE "FlyCAM - Flywheel Control and Monitoring"
+#define MAINWINDOW_ERROR "FlyCAM - Error"
```

Comment:
Consistency, yes, and better error messages. If there were a syntax error in a string that is #defined, the error message would be weird. e.g.,

line 21:
#define ERROR "erro""
...
line 32:
cout << ERROR;

Would give an error like "unterminated string at line 32". But the error is actually at line 21. The error would be more descriptive if it were a string.

So unless there is a definite pro for using the #defines, I'd recommend changing it.